### DISCUSSION TITLE
{Login feature} Repo Download or Installation Issues

### DISCUSSION THREAD

**Discussion by Adrianortiz-PR** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#65] [Reminders]
This forum is set up to help you with any issues you might encounter while downloading and installing the repository. 

**Instructions:**

1. **Download Issues:** If you face any problems while downloading the repo, please describe the issue here. Include any error messages if needed.

2. **Confirmation:** Once you have successfully downloaded and installed the repo, please comment here to confirm. 

**Closure:**

This discussion will be closed once everyone has confirmed successful installation or once all issues have been resolved.

**Comment by Ingrid1089** [comment]
I have successfully downloaded and installed the repository without any issues.

**Comment by GabyMarr** [comment]
I have successfuly cloned the repo without any issues. @Adrianortiz-PR

**Comment by arianrodz21** [comment]
I also finished Svelte and Python Flask Learning.
